### DISCUSSION TITLE
How to save the result

### DISCUSSION THREAD

**Discussion by kgp1202** [b-inary/postflop-solver#34] [Q&A] [answered]
At first, thank you for developing this great solver! I'm very pleasure to use this.

My question is how to save the result of calculations.
Not only the result of one action of OOP or IP, but also the result of calculation within the corresponding turn (flop, turn, river).
Could you please let me know there a good way to save the result?

**Comment by b-inary** [comment, marked as answer]
We can save and load a full tree using experimental file I/O functions (the spec is not yet stabilized, so I cannot guarantee future compatibility), but currently we do not provide an easy way to save or extract a subtree. If saving and loading a full tree is sufficient, please see [src/file.rs](https://github.com/b-inary/postflop-solver/blob/main/src/file.rs) for more information on experimental file I/O functions: basically, the `save_data_to_file` and `load_data_from_file` functions will do the job.

**Reply by kgp1202** [reply]
Thank you!
